44.md

```diff
@@ -8,7 +8,7 @@ Encrypted Payloads (Versioned)
 
 The NIP introduces a versioned encryption data model, allowing multiple algorithm choices to exist simultaneously.
 
-The algorithm described in NIP-04 is potentially vulnerable to [padding oracle attacks](https://en.wikipedia.org/wiki/Padding_oracle_attack) and uses keys which are not indistinguishable from random. For more information, see [here](https://github.com/nostr-protocol/nips/pull/715#issuecomment-1675301250-).
+The algorithm described in NIP-04 is potentially vulnerable to [padding oracle attacks](https://en.wikipedia.org/wiki/Padding_oracle_attack) and uses keys which are not indistinguishable from random. For more information, see [here](https://github.com/nostr-protocol/nips/pull/715#issuecomment-1675301250).
```

Comment:
Don't think it's good idea to link to the comment. Can we simplify it?

> The algorithm described in NIP-04 is not secure. It used bad cryptographic building blocks and must not be used.

Reply:
Your comment was so helpful, but yeah we have it in this PR for reference. I don't think most people realize that NIP 04 compromises your private key.